lib/prismadb.ts

```
@@ -0,0 +1,12 @@
+import {PrismaClient} from "@prisma/client"
+
+// declare global {
+//     let prisma: PrismaClient | undefined
+// }
+
+const globalThis = global as unknown as { prisma: PrismaClient }
```

Biome (1.9.4)

[error] Do not shadow the global "globalThis" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)